mqtt/mqtt-bridge/src/controller.rs

```diff
@@ -111,7 +114,7 @@ impl BridgeController {
                     }
 
                     if let Some(config) = self.bridge_handles.lock().await.get_mut(UPSTREAM) {
-                        if let Err(e) = config.send_update(&bridge_update).await {
+                        if let Err(e) = config.send_update(bridge_update.to_owned()).await {
```

Comment:
had to add to_owned here because it gets consumed

Reply:
which is fine? I guess we don't need `bridge_update` anywhere here
```suggestion
                        if let Err(e) = config.send_update(bridge_update).await {
```

Reply:
Actually I noticed that to_owned is not needed, not sure why I thought was needed.